target/linux/ar71xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -170,8 +209,10 @@ case "$FIRMWARE" in
 	archer-c60-v1|\
 	archer-c60-v2|\
 	cf-e355ac-v2|\
-	cf-e375ac)
+	cf-e375ac|\
+	k2t)
 		ath10kcal_extract "art" 20480 12064
+		ath10kcal_patch_mac_crc $(macaddr_add $(head -c400 $(find_mtd_chardev config) | grep -o wan_mac.* | cut -d\" -f3) +2)
 		ln -sf /lib/firmware/ath10k/pre-cal-pci-0000\:00\:00.0.bin \
```

Comment:
...And now you are patching mac address for all those devices above k2t...

Reply:
it is ok. all work well.

Reply:
You didn't notice what you've done here...
You patched mac read from K2T config partition for all those devices:
```
	archer-c58-v1|\
	archer-c59-v1|\
	archer-c60-v1|\
	archer-c60-v2|\
	cf-e355ac-v2|\
	cf-e375ac|\
	k2t)
```

Reply:
it is a mistake